README.md

```diff
@@ -465,6 +465,37 @@ extensions such as `import com.klaviyo.pushFcm.KlaviyoRemoteMessage.body` to acc
 We also provide an `Intent.appendKlaviyoExtras(RemoteMessage)` extension method, which attaches the data to your
 notification intent that the Klaviyo SDK requires in order to track opens when you call `Klaviyo.handlePush(intent)`.
 
+## In App Forms
+
+### Prerequisites
+- Using version 3.2.0 or higher
+- You need to have Klaviyo `analytics` imported and the company ID must be set
+
+### Setup
+`registerForInAppForms` can be chained with your initialization code to allow for form display.
+```kotlin
+Klaviyo
+    .initialize("KLAVIYO_PUBLIC_API_KEY", applicationContext)
+    .registerForInAppForms()
+```
+You can also call it publicly anywhere you can access the Klaviyo forms module:
+```kotlin
+Klaviyo.registerForInAppForms() // make sure you only call this after initializing Klaviyo company ID
+```
+
+### Behavior
+
+Once you call the `registerForInAppForms` function, the web view will persist unattached to UI until a form is ready to be shown.
```

Comment:
I am going to take another pass over this whole paragraph, it is giving more technical detail than I think is necessary. Like I wouldn't even mention that we're using a webview in the readme for example